docs/tensor_matrix.rst

```diff
@@ -1,41 +1,41 @@
 .. _tensor:
 
-Tensors and matrices
-====================
+Fields and matrices
+===================
```

Comment:
```suggestion
Matrix fields
=============
```
The same to `Vector fields`, it should sounds more than intuitive for STEM students, WDYT? @yuanming-hu @Rullec

Reply:
It's a good idea. But I guess the main task of this PR is simply do replacement, and make sure the replacements have no error.
I would personally prefer to refine the whole documents, make them more fluent and accurate in another PR, HDYT?

Reply:
Yes, thank for the reminder, let's do `matrix fields` iapr.